UI_DSM.Client/Components/NormalUser/Views/PhysicalFlowView.razor.cs

```diff
@@ -144,6 +150,9 @@ protected override void OnInitialized()
 
             this.Diagram.MouseUp += this.Diagram_MouseUp;
             this.Diagram.MouseDoubleClick += this.Diagram_MouseDoubleClick;
+
+            this.disposables.Add(this.WhenAnyValue(x => x.ViewModel.IsOnSavingMode)
```

Comment:
The collection is not dispose, please add `this.disposables.ForEach(x => x.Dispose());` inside the Dispose Method